docs/dafni_cli.md

```diff
@@ -61,6 +75,18 @@ will list all datasets you have access to. Then you can inspect a particular dat
 dafni get dataset <version-id>
 ```
 
+It is also possible to select datasets based on a search term with the following command (the search term will filter datasets based on keywords, as well as the dataset title and description).
+
+```bash
+dafni get datasets --search "Transport"
+```
+
+If you would prefer to output the results in a json file you can use the command
+
+```bash
+dafni get datasets --search "Transport" -j
```

Comment:
This doesn't actually output to a json file, it just outputs it in json format (as apposed to pretty printing it)

Reply:
Fixed now